examples/ksml-data-generator.yaml

```diff
@@ -1,9 +1,74 @@
-  configDirectory: /ksml # When not set defaults to the working directory
-  schemaDirectory: /ksml # When not set defaults to the config directory
-  enableProducers: true
+  configDirectory: /ksml          # When not set defaults to the working directory
+  schemaDirectory: /ksml          # When not set defaults to the config directory
+  storageDirectory: /tmp          # When not set defaults to the default JVM temp directory
+
+  # This section defines if a REST endpoint is opened on the KSML runner, through which
+  # state stores and/or readiness / liveness probes can be accessed.
+  applicationServer:
+    enabled: false                # Set to true to enable, or false to disable
+    host: 0.0.0.0                 # IP address to bind the REST server to
+    port: 8080                    # Port number to listen on
+
+  # This section defines whether a Prometheus endpoint is opened to allow metric scraping.
+  prometheus:
+    enabled: true                 # Set to true to enable, or false to disable
+    host: 0.0.0.0                 # IP address to bind the Prometheus agent server to
+    port: 9999                    # Port number to listen on
+
+  # This section enables error handling or error ignoring for certain types of errors.
+  errorHandling:
+    consume:                      # Error handling definitions for consume errors
+      log: true                   # Log errors true/false
+      logPayload: true            # Upon error, should the payload of the message be dumped to the log file.
+      loggerName: ConsumeError    # Definition of the error logger name.
+      handler: stopOnFail         # How to proceed after encountering the error. Either continueOnFail or stopOnFail.
+    process:
+      log: true                   # Log errors true/false
+      logPayload: true            # Upon error, should the payload of the message be dumped to the log file.
+      loggerName: ProcessError    # Definition of the error logger name.
+      handler: continueOnFail     # How to proceed after encountering the error. Either continueOnFail or stopOnFail.
+    produce:
+      log: true                   # Log errors true/false
+      logPayload: true            # Upon error, should the payload of the message be dumped to the log file.
+      loggerName: ProduceError    # Definition of the error logger name.
+      handler: continueOnFail     # How to proceed after encountering the error. Either continueOnFail or stopOnFail.
+
+  # This section tells KSML with which serializers / deserializers to handle certain notation types.
+  notations:
+    avro:                          # Definition for "avro" notation
+      type: confluent_avro         # For AVRO there are two implementations: apicurio_avro and confluent_avro
+      ## Below this line, specify properties to be passed into Confluent's KafkaAvroSerializer and KafkaAvroDeserializer
+      config:
+        schema.registry.url: https://url.to.schena.registry.org
```

Comment:
Sorry, didn't catch this one before
The examples are meant to be run against the docker compose
These configs should reflect that, right?
So URL `http://schema_registry:8081/apis/ccompat/v7/` for confluent_avro and http://schema_registry:8081/apis/registry/v3` for apicurio